README.md

```diff
@@ -1,3 +1,124 @@
+*Copyright (c) 2024 IHU Liryc, Université de Bordeaux, Inria.*
+
 # MusicBox
 
-*Copyright (c) 2024 IHU Liryc, Université de Bordeaux, Inria.*
+A platform to develop ITK and VTK tools in Python for medInria and associated projects.
+
+## Contents
+
+* [Description](#description)
+* [License](#license)
+* [How to build](#how-to-build)
+  * [Prerequisites](#prerequisites)
+  * [Building the package](#building-the-package)
+  * [Building the standalone](#building-the-standalone)
+* [How to test](#how-to-test)
+* [How to run](#how-to-run)
+* [How to install](#how-to-install)
+
+## Description
+
+The platform consists of:
+
+- a Python package containing the core components, an API for higher-level development, a Qt-based GUI library and a runnable application.
+- an (optional) standalone application that wraps the Python application with an embedded interpreter.
+
+*(This project is still under construction and most of these features are not fully implemented)*
+
+The Python package will be available on PyPi in the near future.
+
+## License
+
+See [LICENSE-BSD-3-Clause](LICENSE-BSD-3-Clause.txt)
+
+## How to build
+
+### Prerequisites
+
+#### Necessary:
+
+Currently ony CMake >= 3.19 is necessary.
+
+#### Optional:
+
+Python >= 3.10 is required if the option to setup a virtual environment in the build tree is chosen.
+
+If the option to build the standalone is chosen, a custom CPython interpreter will be built. The requirements for that can be found [here](https://github.com/python/cpython?tab=readme-ov-file#build-instructions).
+
+#### Building the package:
+
+Using CMake only (from the build directory):
+
+```
+cmake [<options>] <path-to-source-directory>
+cmake --build .
+```
+
+The Python dstribution packages will be located at `<build_directory>/<build_type>/packages/musicbox/dist`
+(When not using a multi-config generator the default build type is `Release`)
+
+To install a virtual environment in the build tree, set the `MBOX_VIRTUAL_ENV` cmake option to `TRUE`.
+
+#### Building the standalone:
+
+Follow the steps to build the Python package, but set the `MBOX_STANDALONE` cmake option to `TRUE`.
+
+(*The standalone is not yet available*)
+
+## How to test:
+
+* Using CTest:
+
+`ctest --verbose`
+
+* Using the virtual environment:
+
+`<build_directory>/<build_type>/venv/bin/python -m musicbox --test all`\
+(on Windows the path is `<build_directory>/<build_type>/venv/Scripts/python.exe`)
+
+* Using the standalone (*not yet available*):
+
+`./musicbox --test all`
+
+* From Python:
+
+```
+from musicbox import dev
+dev.run_tests()
+```
+
+## How to run:
+
+* Using CMake:
+
+```
+cmake --build --target mbox_run_application
+```
+
+* Using the virtual env:
+
+`<build_directory>/<build_type>/venv/bin/python -m musicbox`
+(on Windows the virtual the path is `<build_directory>/<build_type>/venv/Scripts/python.exe`)
```

Comment:
there are maybe too many "the" in the sentence

Reply:
I fixed the typo